content/post/the-future-is-rusty.en.md

```diff
@@ -0,0 +1,69 @@
++++
+title = 'The Future is Rust(y)?'
+slug = 'post'
+image = 'https://cdn.dribbble.com/users/241205/screenshots/4977779/drib_00_webasm_tank.png'

```

Comment:
Wait, if we can have URL's as img sources then why are we even storing images and bloating the repository size. This blog's repo size is right now at 15.5 mb. I think we can do better with this.

Reply:
Well I should have mentioned it. :P